1-js/04-object-basics/06-constructor-new/article.md

```diff
@@ -168,41 +167,41 @@ function SmallUser() {
 alert( new SmallUser().name );  // John
 ```
 
-Usually constructors don't have a `return` statement. Here we mention the special behavior with returning objects mainly for the sake of completeness.
+Geralmente os construtores não têm uma declaração `return`. Aqui, nós mencionamos o comportamento especial com objetos retornados principalmente por uma questão de integralidade.
 
-````smart header="Omitting parentheses"
-By the way, we can omit parentheses after `new`, if it has no arguments:
+````smart header="Omitindo parênteses"
+A propósito, nós podemos omitir os parênteses depois de `new`, se não tiver argumentos.
 
 ```js
-let user = new User; // <-- no parentheses
-// same as
+let user = new User; // <-- sem parênteses
+// igual a
 let user = new User();
 ```
 
-Omitting parentheses here is not considered a "good style", but the syntax is permitted by specification.
+Omitir os parênteses aqui não é considerado uma boa prática, mas a sintaxe é permitida por especificação.
 ````
 
-## Methods in constructor
+## Métodos no construtor
 
-Using constructor functions to create objects gives a great deal of flexibility. The constructor function may have parameters that define how to construct the object, and what to put in it.
+Usar funções construtoras para criar objetos nos dá bastante flexibilidade. A função construtora pode ter parâmetros que definem como construir o objeto, e o quê colocar dentro dele.
 
-Of course, we can add to `this` not only properties, but methods as well.
+Claro, nós podemos adicionar ao `this` não apenas propriedades, mas métodos também.
 
-For instance, `new User(name)` below creates an object with the given `name` and the method `sayHi`:
+Por exemplo, `new User(name)` abaixo, cria um objeto com o `name` dado e o método `sayHi`:
 
 ```js run
 function User(name) {
   this.name = name;
 
   this.sayHi = function() {
-    alert( "My name is: " + this.name );
+    alert( "Meu nome é: " + this.name );
   };
 }
 
 *!*
 let john = new User("John");
 
-john.sayHi(); // My name is: John
+john.sayHi(); // Meu nome é: John
 */!*
 
 /*
```

Comment:
- missing code: available at [Methods in constructor](https://javascript.info/constructor-new#methods-in-constructor)